src/ClientData/include/ClientData/CallstackTypes.h

```diff
@@ -5,12 +5,22 @@
 #ifndef CLIENT_DATA_CALLSTACK_TYPES_H_
```

Comment:
This file would ideally be where `CallstackType` would be if it wasn't a proto, so I would rename it to `CallstackType.h`.